project.clj

```diff
@@ -3,6 +3,16 @@
   :url "https://github.com/weavejester/meta-merge"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.5.1"]]
-  :aliases {"test-all" ["with-profile" "default:+1.6" "test"]}
-  :profiles {:1.6 {:dependencies [[org.clojure/clojure "1.6.0"]]}})
+  :plugins [[lein-cljsbuild "1.0.3"]
+            [lein-doo "0.1.6"]]
+  :aliases {"test-cljs" ["doo" "nashorn" "test" "once"]
+            "test-all" ["do" ["test"] ["test-cljs"]]}
+  :profiles {:provided
+             {:dependencies [[org.clojure/clojure "1.7.0"]
```

Comment:
http://librelist.com/browser//leiningen/2014/9/7/clarifications-about-scope-in-dependencies/#32fcd639eb09c94545bec6a56c200167

I chose profiles over scope due to this mailing list post which mentioned that in leiningen it was removed, I also saw that https://github.com/technomancy/leiningen/issues/741 suggested that using `:scope "provided"` might have some unwanted side-effects.